### DISCUSSION TITLE
[SOLVED] Slider example

### DISCUSSION THREAD

**Discussion by 2ant** [Bodmer/TFT_eSPI#2312] [Q&A - General] [answered]
Hi, when i click on one bar the other one move.

https://user-images.githubusercontent.com/52797081/211372056-e5b7d383-1d68-4f9c-843c-e19abc43df30.mp4

**Comment by Bodmer** [comment, marked as answer]
Enable touch calibration by [setting this line true](https://github.com/Bodmer/TFT_eSPI/blob/master/examples/GUI%20Widgets/Sliders/Slider_demo/Slider_demo.ino#L17).  A LittleFS partition must be present to store the calibration data.

An alternative calibration method to put calibration data in an array instead is in [this library example](https://github.com/Bodmer/TFT_eSPI/blob/master/examples/Generic/Touch_calibrate/Touch_calibrate.ino).

**Comment by yesnoj** [comment]
Hi Bodmer, actually i still can't do it working on my ESP32 S3 devkit, i can't store the values on LittleFS , on serial i receive always the following :

11:41:33.576 -> ./components/esp_littlefs/src/littlefs/lfs.c:1229:error: Corrupted dir pair at {0x0, 0x1}
11:41:33.576 -> E (480) esp_littlefs: mount failed,  (-84)
11:41:33.576 -> E (480) esp_littlefs: Failed to initialize LittleFS
11:41:33.576 -> [   584][E][LittleFS.cpp:95] begin(): Mounting LittleFS failed! Error: -1
11:41:33.608 -> Formating file system
11:41:38.251 -> [  5274][E][vfs_api.cpp:105] open(): /littlefs/TouchCalData1 does not exist, no permits for creation

How can i solve this issue? 
Thanks in advance!

**Comment by raycharleslaw** [comment]
i have the same issue.... the sliders are mixed up.... it works if i redo the calibration.... but the next time it boots it is again messed up... it seems as thougth the calibaration are not being saved or not being recalled??
